### PR TITLE
moved remaining topics

### PR DESCRIPTION
**Pull request by polkhe**
moved tutorial for srna, updated tools for meta and deleted old dockerfile

### CODE REVIEW

File: topics/metagenomics/tutorials/mothur-miseq-sop/tools.yaml [hunk 1]
```diff
@@ -1,7 +1,22 @@
 ---
 api_key: admin
 galaxy_instance: http://localhost:8080
+tool_shed_url: https://toolshed.g2.bx.psu.edu/
```
Comment (bebatut) [inline topics/metagenomics/tutorials/mothur-miseq-sop/tools.yaml]: I think you do not need this line and the two next ones

**Comment by malloryfreeberg** [comment]
@bebatut @polkhe I have not submitted the additional files for the sRNA-seq tutorial yet. I am hoping to get to them this week. Is it better if I submit them before the merge? Please let me know!

**Comment by bebatut** [comment]
@malloryfreeberg do not worry. The idea is to have "empty" files to have the structure. You can fill them later

**Comment by malloryfreeberg** [comment]
@bebatut Ok, then sometime today I can copy over the template "empty" files :)

**Comment by bebatut** [comment]
@polkhe Thanks

**Comment by polkhe** [comment]
@malloryfreeberg no worries, done :) @bebatut sure!

**Pull request merged**